llvm/include/llvm/Object/OffloadBinary.h

```diff
@@ -52,6 +52,13 @@ enum ImageKind : uint16_t {
   IMG_LAST,
 };
 
+/// Flags associated with the Entry.
+enum OffloadEntryFlags : uint32_t {
+  OIF_None = 0,
+  // Entry doesn't contain image. Used to keep metadata only entries.
+  OIF_NoImage = (1 << 0),
```

Comment:
```suggestion
  OIF_Empty = (1 << 0),
```
Or, maybe this or `Metadata`.
```suggestion
  OIF_Notes = (1 << 0),
```